specification.md

```diff
@@ -202,39 +202,38 @@ Event data are stored in CSV files. Simple event data should be
 stored in a single column CSV, with each element in the array indicating the time of the
 event **relative to the start of the dataset**. The first line of the file must contain `start,`,
 indicating that the column contains the times of the event data. Event datasets can be
-distinguished from sampled datasets because the file is a plaintext CSV and `units` attribute must be
```

Comment:
Sampled datasets have a 'units' attribute as well, which currently appears to be null. Keeping this distinction for backwards compatibility makes sense, but if so perhaps `bark:write_sampled` should be changed to disallow 'units' to be anything but None.